vividus-plugin-web-app-to-rest-api/src/main/resources/spring.xml

```diff
@@ -34,7 +40,24 @@
         <property name="seedRelativeUrls" value="${bdd.transformer.headless.seed-relative-urls}" />
     </bean>
 
+    <bean class="org.vividus.http.client.HttpClient" id="resourceValidator"
+        factory-bean="httpClientFactory"
+        factory-method="buildHttpClient">
+        <constructor-arg>
+            <bean parent="restApiHttpClientConfig">
+                <property name="skipResponseEntity" value="true" />
+                <property name="connectionRequestTimeout" value="${resource-checker.connectionRequestTimeout}" />
+                <property name="connectTimeout" value="${resource-checker.connectTimeout}" />
+                <property name="maxTotalConnections" value="${resource-checker.maxTotalConnections}" />
+                <property name="maxConnectionsPerRoute" value="${resource-checker.defaultMaxPerRoute}" />
```

Comment:
what about moving these properties to HTTP client config? they look quite common and may be useful for all users